### ISSUE TITLE
Hint text is small and does not have strong colour contrast

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/platform-forms-client#7900]
## Context

From client: results of their form in fable testing:

> Ran 20 accessibility tests using Fable (10 in English and 10 in French). The accessibility tests mainly focused on screen reader compatibility, with a couple tests using screen magnification tools.

https://cds-snc.freshdesk.com/a/tickets/30088

## Description 

**Helper text being too small and having low colour contrast when using magnification.**

## Findings 

**4 issues related to small and low contrast helper text** (testers used OS Magnification with Safari)

“The descriptive text below the question on this page is more difficult to read as it is smaller and the colour contrast is poor.”

## Examples

> "The colour of the sentence under the question on this page makes the text difficult to read as the colour contrast is poor."

> "The text advising the user to select any option that applies is hard to read as it is in smaller font to the questions or the options provided."

> "Same issue as before, the text under the "Adresse courriel" field is hard to read due to poor colour contrast."

**Relevant WCAG criteria:  1.4.3 Contrast (Minimum)**

**Comment by thiessenp-cds** [comment]
See PR for info on the fix #7937

**Comment by anikbrazeau** [comment]
**Update from Sep 21 stand-up**
> - PR pending review
> - Will flag to Tim when back
